lib/screens/account_settings_screens/account_settings_delete.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:flutter/material.dart';
+import 'package:project_eureka_flutter/screens/login_page.dart';
+import 'package:project_eureka_flutter/services/email_auth.dart';
+
+class AccountSettingsDelete extends StatelessWidget {
+  final EmailAuth _emailAuth = new EmailAuth();
+
+  void deleteUser(context) {
+    _emailAuth.deleteUser().then((_) => Navigator.pushAndRemoveUntil(
```

Comment:
Consider renaming void function, as `deleteUser` can be mixed with the one that Firebase has